docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -214,12 +214,38 @@ offsets as reported by Kafka, the consumer lag per partition, as well as the agg
 consumer lag per partition may be reported as negative values if the supervisor has not received a recent latest offset
 response from Kafka. The aggregate lag value will always be >= 0.
 
+The status report also contains the supervisor's state and a list of recently thrown exceptions (whose max size can be 
+controlled using the `druid.supervisor.maxStoredExceptionEvents` config parameter).  The list of states is as
+follows:
+
+|State|Description|
+|-----|-----------|
+|UNHEALTHY_SUPERVISOR|The supervisor has encountered errors on the past `druid.supervisor.unhealthinessThreshold` iterations|
```

Comment:
Suggest adding some info about the difference between the basic states and the detailed implementation-specific states, and how the detailed implementation states here map to basic states

Reply:
👍